lib/main.js

```diff
@@ -80,13 +81,13 @@ function run() {
             }
         }
         catch (error) {
-            if (error.request.request.retryCount) {
+            if ((_d = (_c = error === null || error === void 0 ? void 0 : error.request) === null || _c === void 0 ? void 0 : _c.request) === null || _d === void 0 ? void 0 : _d.retryCount) {
                 console.log(`request failed after ${error.request.request.retryCount} retries with a delay of ${error.request.request.retryAfter}`);
             }
-            if (!!error.errors && !!error.errors[0] && !!error.errors[0].message && error.errors[0].message.startsWith('No commits between')) {
+            if ((_k = (_j = (_h = ((_e = error === null || error === void 0 ? void 0 : error.errors) !== null && _e !== void 0 ? _e : (_g = (_f = error === null || error === void 0 ? void 0 : error.response) === null || _f === void 0 ? void 0 : _f.data) === null || _g === void 0 ? void 0 : _g.errors)) === null || _h === void 0 ? void 0 : _h[0]) === null || _j === void 0 ? void 0 : _j.message) === null || _k === void 0 ? void 0 : _k.startsWith('No commits between')) {
```

Comment:
fwiw, you can make this a lot more readable with something like:
```js
// `val` if `val` exists else `false`.
function _e (val) { return val !== null && val !== void 0 ? val : false; };

val errors = _e(_e(error).errors) || _e(_e(_e(_e(error).response).data).errors);

if(_e(_e(_e(_e(errors).[0]).message).startsWith('No commits between'))) {
    console.log('No commits between ' + context.repo.owner + ':' + base + ' and ' + owner + ':' + head);
}
```

Reply:
This is transpiled code

Reply:
Ha! I'm a dummy, I didn't even see the .ts right below it.

Reply:
No problem haha :)